tss-esapi/src/structures/buffers.rs

```diff
@@ -220,6 +226,34 @@ pub mod digest {
             Digest(value_as_vec.into())
         }
     }
+
+    macro_rules! impl_from_digest {
+        ($($size:ty),+) => {
+            $(impl From<GenericArray<u8, $size>> for Digest {
+                fn from(value: GenericArray<u8, $size>) -> Self {
+                    Digest(value.as_slice().to_vec().into())
```

Comment:
This is not zeroizing the digest data that you are converting from.

Reply:
good point.